schema.sql

```diff
@@ -41,3 +41,24 @@ CREATE TABLE specializations (
   species_id INTEGER REFERENCES species(id),
   PRIMARY KEY (vet_id, species_id)
 );
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX visits_animal_id_asc ON visits(animal_id ASC);
+CREATE INDEX owners_email_asc ON owners(email ASC);
```

Comment:
Great job team 🤝

- [ ] I can see you have added two indexes for `owners_email_asc` on `email`, and `visits_animal_id` on visits. Kindly also add `CREATE INDEX ON visits (vet_id);`.
- [ ] Also ensure there is an `index-scan` in your optimized output for this query. This shows that you have added an index to optimize your performance.

Reply:
Hello @jlvFlores ✋

- [ ] According to this screenshot, there is actually a performance difference before and after optimization. 
![your-index](https://github.com/zamcham/database-vet-clinic/assets/63917725/bea9cb89-6c7f-4010-b83d-20fd93f2c522)